test/test_automl/automl_utils.py

```diff
@@ -52,13 +60,50 @@ def print_debug_information(automl):
     return os.linesep.join(content)
 
 
+def _includes(scores, all_scores):
+    if len(all_scores) != len(scores):
+        return False
+
+    for score in all_scores:
+        if score not in scores:
+            return False
+
+    return True
+
```

Comment:
Doesn't need to be changed but for your own reference, this function is a pretty common pattern and can be expressed nicely as:
```python
def _include(scores, all_scores)
   return len(set(all_scores) - set(scores)) == 0 
```

The set version isn't the most obvious at a first glance so this version is a bit more clear
```python
def _include(scores, all_scores):
    return all(score in all_scores for score in scores) and len(scores) == len(all_scores)
```